src/components/Logs/Logs.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import md5 from 'md5';
+import { LogsContainerTag, LogMessageTag } from './Logs.theme';
+
+type LogsProps = {
+  messages: string[],
+};
+
+const Logs = ({ messages }: LogsProps) => {
+  return (
+    <LogsContainerTag>
+      { messages.map(message => (
+        <LogMessageTag key={ md5(message) }>{ message }</LogMessageTag>
```

Comment:
maybe u could use `React.Children.toArray`? instead of add extra dependencies

Reply:
@zouxuoz How? Could you write a code example? `message` is an array of strings.